src/components/ResourceList/ResourceList.tsx

```diff
@@ -53,6 +53,12 @@ export interface ResourceListProps {
   /** Item data; each item is passed to renderItem */
   items: Items;
   filterControl?: React.ReactNode;
+  /** The markup to display when no resources exist yet. Renders when set and items is empty. */
+  emptyState?: React.ReactNode;
+  /** The markup to display when no results are returned on search or filter of the list. Renders when `filterControl` is set, items are empty, and `emptyState` is not set.
+   * @default EmptySearchResult
+   */
+  emptySearchState?: React.ReactNode;
```

Comment:
Since these are new props. what will happen to the empty list in web that don't have these props defined?

Reply:
They're optional props so there's no breaking changes with either. Andre added `emptySearchState` because he needed to render custom markup instead of `EmptySearchResult` for the resource list he's working on. Empty state for the list wasn't handled/supported before this PR, the list would just be empty and have no header. But with the new design language rollout we'll be changing empty states for list views from having the large `EmptyState` illustration to having the empty state be in the context of the list so that it's more familiar to merchants once the list has resources.